spec/shared_examples/user_shared_examples.rb

```diff
@@ -527,6 +527,30 @@ def self.matches?(crypted, *tokens)
 
       expect(user.crypted_password).to eq Sorcery::CryptoProviders::BCrypt.encrypt('secret', user.salt)
     end
+
+    it 'if pepper is empty string (default) does not use pepper to encrypt' do
```

Comment:
this case will fail if we add `present?` to [#set_encryption_attributes method](https://github.com/Sorcery/sorcery/blob/ba659b9a8e7564ea6a202a1419445ade3a02697a/lib/sorcery/model.rb#L145) (discussed earlier), as `present?` returns false when `''` is specified, and thus keeps previously-set pepper unchanged.